pt_PT.ini

```diff
@@ -183,48 +183,48 @@ VFPU = VFPU
 
 [Dialog]
 Back = Voltar
-Cancel = Cancel
-ChangingGPUBackends = Changing GPU backends requires PPSSPP to restart. Restart now?
-Choose PPSSPP save folder = Choose PPSSPP save folder
-Confirm Overwrite = Do you want to overwrite the data?
-Confirm Save = Do you want to save this data?
-ConfirmLoad = Load this data?
+Cancel = Cancelar
+ChangingGPUBackends = Mudar de motor de GPU requer que o PPSSPP reinicie. Reiniciar agora?
+Choose PPSSPP save folder = Escolher pasta de gravação do PPSSPP
+Confirm Overwrite = Deseja sobreescrever estes dados?
```

Comment:
Typo – `sobrescrever` is the correct form. Although the translation is correct, I'd change `sobrescrever` to  `substituir` for the sake of clarity.